lattices/src/ord.rs

```diff
@@ -1,13 +1,9 @@
-//! Totally-ordered lattices, [`Max`] and [`Min`].
-//!
-//! Uses [std::cmp::Ord`].
-
 use std::cmp::Ordering;
 
 use super::{ConvertFrom, Merge};
 use crate::LatticeOrd;
 
-/// A totally ordered max lattice. Merging takes the larger value.
+/// A totally ordered max lattice. Merging returns the larger value.
```

Comment:
maybe 'results in' is a bit clearer because calling merge() does not actually return the larger value (merge_owned() does, tho).

Reply:
I like `returns` better since merging refers to the general concept not the specific method imo